back/app/policies/idea_policy.rb

```diff
@@ -82,6 +82,15 @@ def destroy?
   def owner?
     record.author_id == user.id
   end
+
+  def active_moderator?
+    return false unless active? && (admin? || user&.project_moderator?)
+
+    # Needed to permit project moderator to export all ideas, without processing :idea symbols for ideas from unmoderated projects
+    return true if record.instance_of?(Symbol)
```

Comment:
I'm not really happy with this, as it seems like a weird fix.

Because I have introduced a policy that restricts exports for moderators to only ideas that are in projects they moderate, this caused issues with the export of all ideas, as tested in the acceptance test 'when a moderator exports all comments XLSX export' [here](https://github.com/CitizenLabDotCo/citizenlab/blob/dda515a8680e1df1747a54804afe63d1ca4a82f1/back/spec/acceptance/ideas_spec.rb#L486-L507).

The error was:
```Ruby
NoMethodError:
       undefined method `project' for an instance of Symbol
     # ./app/policies/idea_policy.rb:91:in `active_moderator?'
     # ./app/policies/idea_policy.rb:29:in `index_xlsx?
```

I think what was happening is that the idea in the test that is not in a project moderated by the moderator was being represented by a symbol, and therefore the call to `record.project` in the policy `active_moderator?` method would fail.

My fix seem to avoid the issue, without breaking other stuff, but feels like I am missing something simple that could be a more elegant way to avoid the issue.